src/python/pants/backend/python/rules/run_python_binary.py

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
Same changes as the original implementation. This means that `PATH` will be determined by `--pex-executable-search-paths`.